speakers.js

```diff
@@ -45,7 +45,7 @@ export const Sessions = {
                     Retention of DevOps engineers is a difficult challenge for teams. In 2023, 37.5% of the Infragistics Reveal Survey said they expect to find it difficult to find developers in 2023.To keep Software Engineers happy, we need to know how to make them unhappy.
```

Comment:
Do you also need to change this ref to DevOps engineers?

Reply:
Oh yes good spot! I've updated that reference and also corrected a missing full stop. Let me know what you think!